bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT.java

```diff
@@ -178,27 +148,19 @@ public void teardown() throws IOException {
       this.gcsUtil.remove(pathStrs);
     }
 
+    connection.getAdmin().deleteTable(TableName.valueOf(tableId));
     connection.close();
-
-    // delete test table
-    BigtableConfiguration.connect(projectId, instanceId)
-        .getAdmin()
-        .deleteTable(TableName.valueOf(tableId));
   }
 
   private SyncTableOptions createSyncTableOptions() {
     DataflowPipelineOptions syncTableOpts =
         PipelineOptionsFactory.as(DataflowPipelineOptions.class);
-    syncTableOpts.setRunner(DataflowRunner.class);
-    syncTableOpts.setGcpTempLocation(dataflowStagingLocation);
-    syncTableOpts.setNumWorkers(1);
```

Comment:
So we are not explicitly setting the numWorkers now. Just want to check this is a conscious decision. I don't think there will be a problem with not setting it and letting dataflow choose it.

Reply:
yep, I dont think it necessary